kelo_hitbot_driver/MotionCommand command
int32 ovl
float32 blendRadius
uint8 searchFlag
uint8 offsetFlag
kelo_hitbot_driver/Axes dt

